com.github.Desktop.metainfo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.Desktop</id>
```

Comment:
Since this is a fork of the original upstream, it would be better to use an id associated with the fork to not misrepresent it.

Reply:
I will change it